onnx_neural_compressor/algorithms/weight_only/gptq.py

```diff
@@ -318,24 +324,50 @@ def gptq_quantize(
             )
 
             weight_tensor = model.get_initializer(node.input[1])
+            org_shape = weight.shape
             init_share_num = model.get_initializer_share_num(node.input[1])
 
-            satisfy_MatMulNBits_condition = Version(ort.__version__) > constants.ONNXRT1161_VERSION and num_bits == 4
+            satisfy_MatMulNBits_condition = ort_version > constants.ONNXRT1161_VERSION and num_bits == 4
```

Comment:
it is better to put such a ORT version check for both MatMulNBits and standard DQ out of algorithm specific logic/file.  maybe in high level API layer by detecting quant_format